evoting/service/service.go

```diff
@@ -587,13 +590,18 @@ func (s *Service) load() error {
 	return nil
 }
 
+func (s *Service) db() *skipchain.SkipBlockDB {
```

Comment:
This seems to not be used?

Reply:
Aha, looks like I copied this in haste from ocs and forgot to remove it